tests/test_datasets/test_dataset.py

```diff
@@ -350,7 +352,36 @@ def test_get_sparse_categorical_data_id_395(self):
         self.assertEqual(len(feature.nominal_values), 25)
 
 
-class OpenMLDatasetQualityTest(TestBase):
+class OpenMLDatasetFunctionTest(TestBase):
+    @unittest.mock.patch("openml.datasets.dataset.pickle")
+    def test__read_features(self, pickle_mock):
```

Comment:
Some better description of the test case would be welcome (also for the one below).
I think what is tested here is that the fallback behavior is the features pickle file is not present?

Reply:
Done.